TheEthicalHackerCup/Assets/Scenes/AttackMenu.unity

```diff
@@ -2537,43 +2686,6 @@ PrefabInstance:
       objectReference: {fileID: -9125669662464014136, guid: 3f5cd0f991a440e4c958d1939cb29910, type: 2}
     m_RemovedComponents: []
   m_SourcePrefab: {fileID: 100100000, guid: 67782a0a5b8eb7b4f9810af895a44bff, type: 3}
---- !u!1 &1752556128
-GameObject:
-  m_ObjectHideFlags: 0
-  m_CorrespondingSourceObject: {fileID: 0}
-  m_PrefabInstance: {fileID: 0}
-  m_PrefabAsset: {fileID: 0}
-  serializedVersion: 6
-  m_Component:
-  - component: {fileID: 1752556129}
-  m_Layer: 5
-  m_Name: TEMP STUFF
-  m_TagString: Untagged
-  m_Icon: {fileID: 0}
-  m_NavMeshLayer: 0
-  m_StaticEditorFlags: 0
-  m_IsActive: 1
---- !u!224 &1752556129
-RectTransform:
```

Comment:
unrelated to line, but I noticed that the attack tutorial text box is a bit cut off.
![image](https://user-images.githubusercontent.com/72260849/206343851-1f6b3faf-53ec-4fe3-ab57-944c9686ecbc.png)